client/src/pages/Login/LoginForm/LoginForm.tsx

```diff
@@ -46,9 +46,11 @@ export default function Login({ handleSubmit }: Props): JSX.Element {
     >
       {({ handleSubmit, handleChange, values, touched, errors, isSubmitting }) => (
         <form onSubmit={handleSubmit} className={classes.form} noValidate>
+          <FormHelperText id="email-helper" className={classes.label}>
+            Your e-mail address:
+          </FormHelperText>
```

Comment:
On my screen there's not a lot of room between the label and input, maybe you can add some more margin

Reply:
Added more margin between label and inputs.